ecosystem/sep-0031.md

```diff
@@ -192,17 +190,14 @@ Content-Type: application/json
   "asset_issuer": "GDRHDSTZ4PK6VI3WL224XBJFEB6CUXQESTQPXYIB3KGITRLL7XVE4NWV",
   "fields": {
     "sender": {
-      "first_name": "Alice",
-      "last_name": "Jones"
+       "id": 123
```

Comment:
maybe add a little note here like `// This is the customer ID returned from SEP12`